plugins/modules/dcnm_vrf.py

```diff
@@ -809,16 +907,32 @@ def diff_for_attach_deploy(self, want_a, have_a, replace=False):
                         break
 
             if not found:
-                if bool(want["isAttached"]):
-                    del want["isAttached"]
-                    want["deployment"] = True
-                    attach_list.append(want)
-                    if bool(want["is_deploy"]):
-                        dep_vrf = True
+                try:
+                    if want["isAttached"]:
```

Comment:
Should you be using `self.to_bool()` here also?

```python
try:
    if self.to_bool("isAttached", want):
...
```

Reply:
Yes!  I'll change it.  Thank you!